railib/api.py

```diff
@@ -136,11 +136,12 @@ class Permission(str, Enum):
 # Context contains the state required to make rAI API calls.
 class Context(rest.Context):
     def __init__(self, host: str = None, port: str = None, scheme: str = None,
-                 region: str = None, credentials=None):
+                 region: str = None, credentials=None, auth0_audience: str = None):
         super().__init__(region=region, credentials=credentials)
         self.host = host
         self.port = port or "443"
         self.scheme = scheme or "https"
+        self.auth0_audience = auth0_audience
```

Comment:
can we just call this "audience" .. i believe this is an OAuth concept and not specific to auth0.